src/config/Config.zig

```diff
@@ -1113,9 +1113,12 @@ keybind: Keybinds = .{},
 /// to be a certain position to fit within the grid. There is nothing Ghostty
 /// will do about this, but it will make an effort.
 ///
+/// Important note: Setting this value will only work on macOs and glfw builds
+/// on Linux. GTK 4.0 does not support setting the window position.
+///
 /// This will default to the top-left corner of the screen if not set (0, 0).
-@"window-position-x": i32 = 0,
-@"window-position-y": i32 = 0,
+@"start-position-x": i32 = 0,
+@"start-position-y": i32 = 0,
 
```

Comment:
`window-initial-position-x` and `window-initial-position-y` I think would be clearer as to their purpose.

Reply:
cool sounds good, I'll revert that change

Reply:
The should also be unsigned and optional. That way you can tell if the user did not set these in their config. 32 bits is probably too much, I think that a `u16` should be fine.

Reply:
Correct me if I'm wrong but you can't you have negative coordinates depending on your monitor layout? I might just be mixing this up with something else but I thought that was a possibility

Reply:
I can't say for 100% sure, you'd need to check the documentation of the various APIs used to position the window. We definitely want these to be optional though. If the user didn't set them in the config, we don't want to force them into a specific location.

Reply:
I'm going to drop this here for later but macOS allows negative values (my understanding is that this value is based on your primary monitor's bottom left corner). I'll also dig into the Linux side later to see if I can find any docs supporting this behaviour.

https://developer.apple.com/documentation/appkit/nswindow/setframeorigin(_:)#Discussion